derive/src/lib.rs

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#![cfg_attr(not(feature = "std"), no_std)]
+// #![cfg_attr(not(feature = "std"), no_std)]
```

Comment:
?

Reply:
Ouch, my bad.

But while we're at it: we should probably remove this anyway. The derive crate doesn't need to be no-std I think (only its output must be no-std). And as we're using `proc-macro-crate` already it's perhaps good to acknowledge the fact that `scale-info-derive` isn't actually no-std?

Reply:
Yeah I'm not sure why that's there in the first place, but if we remove it should not be hidden in this PR

Reply:
See https://github.com/paritytech/scale-info/pull/69#discussion_r587434349 – I removed it in that PR.